Add description metadata (#1659)
ezsystems/developer-documentation: docs/guide/content_model.md

---

> description: Ibexa DXP's content model relies on Content items based on Content Types, containing different content Fields.

Author: ```suggestion
description: Ibexa DXP's content model relies on Content items that are instances of Content Types and contain content Fields.
```
